packages/features/redis/IRedisService.d.ts

```diff
@@ -0,0 +1,15 @@
+export interface IRedisService {
```

Comment:
Why do we even need this file? I just tried having RedisService.ts without `implements` and type-check passes.

Reply:
Makes sense !